src/main/java/com/hivemq/client/internal/mqtt/MqttClientSslConfigImpl.java

```diff
@@ -113,7 +137,8 @@ public boolean equals(final @Nullable Object o) {
         return Objects.equals(keyManagerFactory, that.keyManagerFactory) &&
                 Objects.equals(trustManagerFactory, that.trustManagerFactory) &&
                 Objects.equals(cipherSuites, that.cipherSuites) && Objects.equals(protocols, that.protocols) &&
-                (handshakeTimeoutMs == that.handshakeTimeoutMs);
+                (handshakeTimeoutMs == that.handshakeTimeoutMs) &&
+                Objects.equals(hostnameVerifier, that.hostnameVerifier);
```

Comment:
Is the equals implementation used anywhere? Since a lot of the fields don't implement equals them selfs, this seems somewhat pointless.

Reply:
I guess its still nice to have. Never mind.

Reply:
It is still good that 2 objects with the same references are equal